src/main/java/team/ebi/epicbanitem/check/Triggers.java

```diff
@@ -46,7 +49,10 @@
     private static final SortedMap<String, CheckRuleTrigger> triggers = new TreeMap<>();
 
     @Inject
-    public Triggers(EventManager eventManager, PluginContainer pluginContainer) {
+    public Triggers(EventManager eventManager, PluginContainer pluginContainer, ThrowListener throwListener, DropListener dropListener) {
```

Comment:
For what?

Reply:
Register listeners.
Split code to modules

Reply:
But should it be here?

Reply:
Why not? Modular